### ISSUE TITLE
⚠ WARN: Cannot read property 'length' of null

### ISSUE THREAD

**Issue by salomvary** [assetgraph/assetgraph-builder#660]
I found this weird edge case:

index.html:
```
<script nobundle>
  foo('/icon.png'.toString('url'))
</script>
```

Command:
```
npx buildProduction  --stoponwarning  --outroot dist  index.html
```

Output:
```
[...]
 ✔ 0.000 secs: mergeIdenticalAssets
 ✔ 0.002 secs: replaceSymbolsInJavaScript
 ⚠ WARN: Cannot read property 'length' of null
 ✘ ERROR: compressJavaScript: A warning was encountered while stopOnWarning is on, exiting with a non-zero exit code
```

Workaround:
- Remove `nobundle` or
- Change `/icon.png` to `icon.png`

**Comment by papandreou** [comment]
Thanks for the report! Fixed in 6.6.4.

**Comment by salomvary** [comment]
Amazing, thanks!
